src/tests/system/tests/test_default_debug_level.py

```diff
@@ -0,0 +1,196 @@
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_default_debug_level__check_with_login(client: Client, provider: GenericProvider):
+    """
+    :title: Successful login with default debug level doesn't generate any logs
+    :setup:
+        1. Add user to SSSD, set its password
+        2. Add fallback_homedir (generates extra logs on
+            user auth if not specified)
+    :steps:
+        1. Clear cache and logs
+        2. Start SSSD with default debug level
+        3. Check total log size before user authentication
+        4. Authenticate with user
+        5. Check total log size after authentication
+    :expectedresults:
+        1. Logs cleared
+        2. SSSD started successfully
+        3. Checked correctly
+        4. User is authenticated
+        5. Log sizes before and after are same
+    :customerscenario: False
+    """
+
+    def log_length() -> int:
+        try:
+            output = client.fs.wc("/var/log/sssd/*", lines=True).stdout
+            return int(output.split()[0])
+        except SSHProcessError:
+            return 0
+
+    provider.user("user1").add(password="Secret123")
+    client.sssd.common.local()
+    client.sssd.default_domain = "local"
+    client.sssd.domain["fallback_homedir"] = "/home/%%u"
+
+    client.sssd.clear(db=True, memcache=True, logs=True, config=False)
+    client.sssd.start(debug_level=None)
+
+    start_log_length = log_length()
+    assert client.auth.ssh.password("user1", "Secret123"), "Authentication failed"
+    assert start_log_length == log_length(), "Debug messages were generated"
+
+
+@pytest.mark.ticket(bz=1893159)
+@pytest.mark.topology(KnownTopology.Client)
+def test_default_debug_level__fatal_and_critical_failures(client: Client):
+    """
+    :title: Check that messages with levels 0 and 1 are logged
+    :setup:
+        1. Start SSSD with default debug level (config file is created)
+    :steps:
+        1. Restrict sssd.conf permissions
+        2. Restart sssd
+        3. Check logs
+    :expectedresults:
+        1. Permissions changed successfully
+        2. SSSD fails to start
+        3. Fatal failures (level 0) and Critical failures (level 1) are in log file
+    :customerscenario: True
+    """
+    client.sssd.common.local()
+    client.sssd.default_domain = "local"
+    client.sssd.start(debug_level=None)
+
+    assert client.host.ssh.run("chmod 444 /etc/sssd/sssd.conf").rc == 0
+    assert client.sssd.restart(debug_level=None, raise_on_error=False, apply_config=False).rc != 0
+
+    log_str = client.fs.read("/var/log/sssd/sssd.log")
+    assert "0x0010" in log_str, "Fatal failures were not found in log"
+    assert "0x0020" in log_str, "Critical failures were not found in log"
+
+
+@pytest.mark.ticket(bz=1893159)
+@pytest.mark.topology(KnownTopology.Client)
+def test_default_debug_level__cannot_load_sssd_config(client: Client):
+    """
+    :title: default debug logs: Check default debug level 2
```

Comment:
I'd change this title to `Check that messages with level 2 are logged when SSSD can't load config`

Reply:
Yes